doc/nrf/libraries/networking/downloader.rst

```diff
@@ -187,12 +187,30 @@ The following snippet shows how to download a file using HTTPS:
 Limitations
 ***********
 
-The library requires the host server to provide a Content-Range field in the HTTP GET response header when using HTTPS with the nRF91 Series devices.
-If this header field is missing, the library logs the following error::
+The following limitations apply to this library:
 
-   <err> downloader: Server did not send "Content-Range" in response
+nRF91 Series TLS limitation
+===========================
 
- Due to internal limitations, maximum CoAP block size is 512 bytes.
+The nRF91 Series modem has a size limit of ~2 kB for receiving TLS packages.
```

Comment:
There is some inconsistent information, in some places it says `~2 kB`, in other places `2 kB`. Barring the fact that kB means 1000 bytes and not 1024, to prevent ambiguity I would write the exact number of bytes (ex. `2048`).

Reply:
We don't have a strict number as it is dependent on other factors in the modem. I'll add a note referring to the changelog and avoid using an exact number.